### DISCUSSION TITLE
About downloading the code

### DISCUSSION THREAD

**Discussion by smallrainbow** [epochpic/epoch#270] [Q&A] [answered]
Dear developers,

![image](https://user-images.githubusercontent.com/13895919/129719358-6ca7b30a-d0e7-4aba-a385-34b94d550b96.png)
When I download the epoch code by clicking it as shown in the screenshot above, why the downloaded folder SDF is empty?
How can I fix it, please? Thank you!

**Comment by TomGoffrey** [comment, marked as answer]
This is explained in the first line of the README file, which you see on the EPOCH GitHub webpage. Please follow the instructions there - either download from the releases page, or clone the repository with the `--recursive` flag.

**Comment by smallrainbow** [comment]
Thank you!

Tom Goffrey ***@***.***> 于2021年8月17日周二 下午7:47写道：
